### ISSUE TITLE
No good walk at 15hrs

### ISSUE THREAD

**Issue by chaneldobler** [CMM-25/a3#27]
Dear TA's,

I'm at 14.85 hours and I don't have a good run yet. I have tried many different sigmas and so far none of them have a good result. Is there a way to get more time? I didn't start the cartwheel training yet and I have doubts whether I can finish the walk in the 20 hrs. I promise I was responsible with my time and I have many different runs with different results but I have nothing I can hand in yet. Any help is appreciated.

Best,

Chanel

**Comment by chaneldobler** [comment]
This is my best walk to show what I mean with not good enough: 

https://github.com/user-attachments/assets/23f0e30c-7dd8-4149-a087-5c5b76e69aca
![Image](https://github.com/user-attachments/assets/245d98af-611c-452e-9b35-0e037b210e2f)

I'm now at 17.67 hrs. Should I just stop and leave it.

**Comment by fzargarbashi** [comment]
Hi,
I understand you're having trouble solving the problem within the coordinated time. Unfortunately, without a specific question or more detail about where you're getting stuck, it's hard for me to provide targeted help. From the video it seems the motion is jittering for which I can recommend strengthening the joint_target_rate reward. If you have a specific question, I'd be happy to help further.

I’ll check whether we have the capacity to increase the compute time, but I can’t make any promises.

**Comment by chaneldobler** [comment]
Dear Fatemeh,

Thank you for your feedback. I feel I explored different settings for the hyperparameters but could definitely still benefit from trying more. Thank you for the hint with the target joint rate. I remember people saying they dropped it out completely so I set it quite high too. I will try one last run in my remaining time where I tighten the target joint rate and hope that will improve my walk to where I feel confident handing it in. Thank you very much for checking out my options for the time. I apologize for making circumstances once again. I promise I try my best.

**Comment by chaneldobler** [comment]
A note to the joint target rate: I just tried it again with the default sigma 1 or even 10 and I observe that it pulls all my rewards towards 0 where my rewards kind of steadily grow with a sigma of 50 for joint target rate.
 joint target rate sigma 50: (run eval posted above)
![Image](https://github.com/user-attachments/assets/28f42cf9-3201-4bfd-aace-3e3cecdf63c9)
joint target rate sigma 10:
![Image](https://github.com/user-attachments/assets/b45932c7-9a1b-4b72-9ae3-88a5d869234c)


What I would do in a situation like this is trying a bunch of different sigmas for the joint rate until I have the smallest one that doesn't drive all my rewards to 0. I'm not sure that's what I'm supposed to do but that's what I've been doing with the different rewards sigmas just lowering ones where they go towards 0 after very few iterations and increasing sigmas where the reward doesn't seem to grow. I'm not sure that's what I'm supposed to do but it was my approach to this tuning. It just means I spent so much time trying so many different values.

**Comment by fzargarbashi** [comment]
Playing just with the reward sigmas might not be enough. If the training is unstable (noise getting higher during training instead of decreasing) then you could try lowering the learning rate for a more stable training.
